content/panorama/scripts/hotkey_tracker.js

```diff
@@ -1,8 +1,8 @@
-'use strict';
+Game.Events = {};
 
 function GetKeybind(key) {
-	var keyElement = $.GetContextPanel().FindChildTraverse(key)
-	keyElement.visible = false;
+	var keyElement = $('#' + key)
+	keyElement.visibility = "collapse"
```

Comment:
I meant that you can move it to styles file
https://github.com/ark120202/angel-arena-black-star/blob/develop/content/panorama_src/styles/arena_setup.sass